matchms/similarity/__init__.py

```diff
@@ -1,4 +1,17 @@
-"""similarity module"""
+"""
+Functions for computing spectra similarities
+############################################
+
+Matchms provides a number of frequently used similarity scores to compare mass
+spectra. This includes scores based on comparing peak positions and intensities
```

Comment:
To make easier to read can this be reformatted into a bullet list?

Reply:
Done (need to check if it renders fine).

Reply:
Renders ok